src/gd_avif.c

```diff
@@ -165,6 +165,9 @@ static avifBool isAvifError(avifResult result, const char *msg) {
 
 	We ignore readFlags, just as the avifIO*ReaderRead() functions do.
 
+	In libavif 0.8.2, it's normal to get 0 bytes on the last read.
```

Comment:
The comment at line 166 is incorrect. The avifIO*ReaderRead() functions require readFlags to be zero:

```
    if (readFlags != 0) {
        // Unsupported readFlags
        return AVIF_RESULT_IO_ERROR;
    }
```

Reply:
I see your point. I thought of `readFlags` as simply a thing to ignore, but I see that the functions in `libavif/io.c` consistently throw an error. I should do the same.